support-frontend/app/controllers/LandingCopyProvider.scala

```diff
@@ -0,0 +1,31 @@
+package controllers
+
+import com.gu.i18n.Country.UK
+import com.gu.i18n.{Country, CountryGroup}
+import com.gu.support.catalog.Product
+import com.gu.support.config.Stage
+import com.gu.support.promotions.{ProductPromotionCopy, PromoCode, PromotionCopy, PromotionServiceProvider}
+
+class LandingCopyProvider(
+  promotionServiceProvider: PromotionServiceProvider,
+  stage: Stage
+) {
+
+  // To see if there is any promotional copy in place for this page we need to get a country in the current region (country group)
+  // this is because promotions apply to countries not regions. We can use any country however because the promo tool UI only deals
+  // with regions and then adds all the countries for that region to the promotion
```

Comment:
this comment was in the "paper" version of the code, and looked useful, so I retained it.